packages/eas-cli/src/commands/update/edit.ts

```diff
@@ -0,0 +1,142 @@
+import { Flags } from '@oclif/core';
+import assert from 'assert';
+import chalk from 'chalk';
+
+import EasCommand from '../../commandUtils/EasCommand';
+import { EasNonInteractiveAndJsonFlags } from '../../commandUtils/flags';
+import { PublishMutation } from '../../graphql/mutations/PublishMutation';
+import { UpdateQuery } from '../../graphql/queries/UpdateQuery';
+import Log from '../../log';
+import { promptAsync } from '../../prompts';
+import {
+  formatUpdateGroup,
+  getUpdateGroupDescriptions,
+  getUpdateJsonInfosForUpdates,
+} from '../../update/utils';
+import { enableJsonOutput, printJsonOnlyOutput } from '../../utils/json';
+
+export default class UpdateEdit extends EasCommand {
+  static override description = 'edit all the updates in an update group';
```

Comment:
same reasoning as https://github.com/expo/eas-cli/pull/2502#discussion_r1727509371
```suggestion
  static override hidden = true;
  static override description = 'edit all the updates in an update group';
```